surfer/tests/test_viz.py

```diff
@@ -75,10 +75,13 @@ def test_image():
     brain = Brain(*std_args, size=100)
     brain.save_image(tmp_name)
     brain.save_image(tmp_name, 'rgba', True)
-    brain.save_montage(tmp_name, ['l', 'v', 'm'], orientation='v')
-    brain.save_montage(tmp_name, ['l', 'v', 'm'], orientation='h')
-    brain.save_montage(tmp_name, [['l', 'v'], ['m', 'f']])
     brain.screenshot()
+    if not sys.platform.startswith('linux') or \
+            os.getenv('TRAVIS', 'false') != 'true':
```

Comment:
so this will not run on any linux system ?

Reply:
It runs fine on my local Linux, I'm assuming it's an Xvfb / temporary Mayavi bug

Reply:
if you're asking about the conditional -- these lines will run if the system is not linux or it's not travis (same as `not (linux and travis)`)